### DISCUSSION TITLE
Globally configure options not related to styling

### DISCUSSION THREAD

**Discussion by bricked-dev** [nix-community/stylix#641] [Ideas]
Stylix is great, because it takes a simple palette and applies it to many targets even if these don't support a central configuration. This could also be applied to locales, keyboard configuration, key combos, etc. For example, there are separate keyboard layout options for Xserver, the console and Hyprland. Allthough the use case is limited and perhaps out of scope, it would be the same idea as with styling.

**Comment by trueNAHO** [comment]
> This could also be applied to locales, keyboard configuration, key combos, etc. For example, there are separate keyboard layout options for Xserver, the console and Hyprland.

There have already been some discussions on expanding Stylix's scope to also define font-related options and minor things like border widths. However, your requested features probably go beyond Stylix's scope and should be implemented in your central NixOS or Home Manager configuration or upstreamed to NixOS or Home Manager.
